Gemfile

```diff
@@ -2,8 +2,8 @@ source "https://rubygems.org"
 
 gem "mdl", "~> 0.3.1"
 gem "posix-spawn"
-gem "rake"
 
-group :development do
-  gem "rspec"
-end
+gem "rake"
+gem "rspec"
+gem "simplecov"
+gem "codeclimate-test-reporter", "1.0.0.pre.rc2"
```

Comment:
Ah, right

I had some builds failing with an error about `bundle exec codeclimate-test-reporter` command not being found and my first theory was that the gem wasn't being installed. That turned out not to be right -- I actually needed to switch from `docker run ... bundle exec rake && bundle exec codeclimate-test-reporter` to `docker run ... sh -c "bundle exec rake && bundle exec codeclimate-test-reporter` to allow that compound expression.

I can put it back in a group, although I'm not sure what value it's adding. We don't do `bundle install --without development` in the Dockerfile, or anything like that (and couldn't, because we want these dependencies in docker so we can run our tests inside docker). It doesn't hurt, and I suppose makes it easier to understand the purpose of a given dependency, so I guess it's fine and I'll add it back